src/earn/EarnPoolInfoScreen.test.tsx

```diff
@@ -7,20 +7,27 @@ import EarnPoolInfoScreen from 'src/earn/EarnPoolInfoScreen'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { navigateToURI } from 'src/utils/linking'
+import networkConfig from 'src/web3/networkConfig'
 import MockedNavigator from 'test/MockedNavigator'
 import { createMockStore, getMockStackScreenProps } from 'test/utils'
-import { mockEarnPositions } from 'test/values'
+import { mockArbUsdcTokenId, mockEarnPositions, mockTokenBalances } from 'test/values'
 
-const defaultStore = createMockStore({})
+const mockPoolTokenId = networkConfig.aaveArbUsdcTokenId
```

Comment:
can we get this from mockEarnPositions? we are moving away from using networkConfig with the multi pool work

Reply:
Changed in /3bf06c071bd628289bfe6b16f48ad3b38f6fc613!